projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -48,6 +48,23 @@ public function reregister_menu_items() {
 		ksort( $GLOBALS['menu'] );
 	}
 
+	/**
+	 * Returns the Calypso origin.
+	 *
+	 * @return string
+	 */
+	private function get_calypso_origin() {
```

Comment:
It's related to the Calypso, I think we can keep using `https://wordpress.com` for now and continue to migrate to jetpack-mu-wpcom plugin in the follow-up PR. WDYT?

Reply:
@arthur791004 Do you mean that we should move both `site_editor_dashboard_link` and `site_editor_theme_preview_back_link` into `jetpack-mu-wpcom` as well as `get_calypso_origin`? 👀

Reply:
YES!

Reply:
That makes sense then 🙂 Let me fix it!

Reply:
I addressed it and added a PR description.